resources/configurations/examples/wan_server.yaml

```diff
@@ -28,14 +28,9 @@ WANServer:                                                        # 6
   type: wan                                                       # 7
   id: 1                                                           # 8
   listening-addresses:                                            # 9
-  [
-    {
-      ip: "1.1.1.1",                                              # 10
-      port: 11666,                                                # 11
+    - ip: "1.1.1.1"                                               # 10
+      port: 11666                                                 # 11
       transport: "udp"                                            # 12
-    },                                                            # 13
-  ]
-
 
 ##################################
 # CONFIGURATION DESCRIPTION
```

Comment:
Change the phrase `the behavior of each of the routers do not change.` to say `it is the same` or somehow explain that is independetly if you are Client or Server.
Now it looks like it says it does not change, but are not equal.

Reply:
:+1: